crates/bevy_core_pipeline/src/postprocess/postprocess.wgsl

```diff
@@ -0,0 +1,81 @@
+// Miscellaneous postprocessing effects, currently just chromatic aberration.
+
+#import bevy_core_pipeline::fullscreen_vertex_shader::FullscreenVertexOutput
+
+// See `bevy_core_pipeline::postprocess::ChromaticAberration` for more
+// information on these fields.
+struct PostprocessingSettings {
+    chromatic_aberration_intensity: f32,
+    chromatic_aberration_max_samples: u32,
+    unused_a: u32,
+    unused_b: u32,
+}
+
+// The source framebuffer texture.
+@group(0) @binding(0) var source_texture: texture_2d<f32>;
+// The sampler used to sample the source framebuffer texture.
+@group(0) @binding(1) var source_sampler: sampler;
+// The 1D lookup table for chromatic aberration.
+@group(0) @binding(2) var chromatic_aberration_lut_texture: texture_1d<f32>;
+// The sampler used to sample that lookup table.
+@group(0) @binding(3) var chromatic_aberration_lut_sampler: sampler;
+// The settings supplied by the developer.
+@group(0) @binding(4) var<uniform> settings: PostprocessingSettings;
+
+@fragment
+fn fragment_main(in: FullscreenVertexOutput) -> @location(0) vec4<f32> {
```

Comment:
It doesn't matter right now, but once we start having multiple effects, I think we should consider making them reusable by putting each effect in a separate file with it's own bindings. That way people could import the shader file in their own pipeline and bind the required data manually. Not required for this PR though since it's already the only effect anyway.

Reply:
Done.